dev/tools/merge-pr.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/sh -e
+
+# This script depends (at least) on git and jq.
+# It should be used like this: dev/tools/merge-pr.sh /PR number/
+
+#TODO: check arguments and show usage if relevant
+
+PR=$1
+
+git fetch github
+
+BASE_BRANCH=`curl https://api.github.com/repos/coq/coq/pulls/$PR 2>/dev/null | jq -r '.base.label'`
+LOCAL_BASE_BRANCH=`git rev-parse --abbrev-ref HEAD`
+
+COMMIT=`git rev-parse github/pr/$PR`
+STATUS=`curl https://api.github.com/repos/coq/coq/commits/$COMMIT/status 2>/dev/null | jq -r '.state'`
+
+if [ $BASE_BRANCH != "coq:$LOCAL_BASE_BRANCH" ]; then
+  echo "Wrong base branch"
+  read -p "Bypass? [y/n] " -n 1 -r
+  echo
+  if [[ ! $REPLY =~ ^[Yy]$ ]]
+  then
+    exit 1
+  fi
+fi;
+
+if [ $STATUS != "success" ]; then
+  echo "CI status is \"$STATUS\""
+  read -p "Bypass? [y/n] " -n 1 -r
+  echo
+  if [[ ! $REPLY =~ ^[Yy]$ ]]
+  then
+    exit 1
+  fi
+fi;
+
+git merge --no-ff github/pr/$PR -m "Merge PR #$PR: `curl https://api.github.com/repos/coq/coq/pulls/$PR 2>/dev/null | jq -r '.title'`" -e
```

Comment:
Here, the use of an explicit `-S` would remind anyone using this script to sign the merge commit.